install/updateVersion_orig.sh

```diff
@@ -274,11 +274,8 @@ function update_istio_addons() {
   mkdir -p $DEST
   pushd $TEMP_DIR/templates/addons
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" grafana.yaml.tmpl
-  execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" servicegraph.yaml.tmpl
   sed "s|{ISTIO_NAMESPACE}|${ISTIO_NAMESPACE}|" grafana.yaml.tmpl  > $DEST/grafana.yaml
```

Comment:
is this what was needed for one of the ci/cd test cases?

Would be nice if we could remove updateVersion_orig.sh in master once 0.8 unfreezes.

Reply:
Agreed, but there are still tests that depend on stuff.